lib/index.js

```diff
@@ -45,18 +45,19 @@
  * The main interface is the {@link analyze} function.
  * @module escope
  */
-"use strict";
-
 /* eslint no-underscore-dangle: ["error", { "allow": ["__currentScope"] }] */
 
-const assert = require("assert");
+import { readFileSync } from "fs";
+import assert from "assert";
+
+import ScopeManager from "./scope-manager.js";
+import Referencer from "./referencer.js";
+import Reference from "./reference.js";
+import Variable from "./variable.js";
 
-const ScopeManager = require("./scope-manager");
-const Referencer = require("./referencer");
-const Reference = require("./reference");
-const Variable = require("./variable");
-const Scope = require("./scope").Scope;
-const version = require("../package.json").version;
+const { version } = JSON.parse(
```

Comment:
As @mdjermanovic mentioned, this change won’t work in a browser, where ESLint currently works for our demo (and probably for others too). If version is needed, we may need to do something like what we did in the Espree repo to work around this limitation.

Reply:
I have made a change akin to that in Espree. Let me know if you want to drop the `version` export entirely. We are currently exporting `package.json` in `exports` so the user should be able to get at it.

Reply:
`version` is used in `vue-eslint-parser` [here](https://github.com/vuejs/vue-eslint-parser/blob/66d424ecc108c8850c4ff225fea0199cfc8eee6b/src/common/eslint-scope.ts#L18)